title_screen.gd

```diff
@@ -0,0 +1,13 @@
+extends Control
+
+
+func _on_play_pressed() -> void:
+	get_tree().change_scene_to_file("res://level.tscn")
```

Comment:
We should allow the level to be set as an @export variable. The type could be `PackedScene` so that it's ensured it's a scene rather than any file path.